<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmakeProjectManager::Internal::ClassDefinition</class>
 <widget class="QTabWidget" name="QmakeProjectManager::Internal::ClassDefinition">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="whatsThis">
   <string>The header file</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="sourceTab">
   <attribute name="title">
    <string>&amp;Sources</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="widgetLibraryLabel">
        <property name="text">
         <string>Widget librar&amp;y:</string>
        </property>
        <property name="buddy">
         <cstring>widgetLibraryEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="widgetLibraryEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="widgetProjectLabel">
        <property name="text">
         <string>Widget project &amp;file:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>widgetProjectEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="widgetProjectEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="widgetHeaderLabel">
        <property name="text">
         <string>Widget h&amp;eader file:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>widgetHeaderEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="widgetHeaderEdit">
        <property name="whatsThis">
         <string>The header file has to be specified in source code.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="widgetSourceLabel">
        <property name="text">
         <string>Widge&amp;t source file:</string>
        </property>
        <property name="buddy">
         <cstring>widgetSourceEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="widgetSourceEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="widgetBaseClassLabel">
        <property name="text">
         <string>Widget &amp;base class:</string>
        </property>
        <property name="buddy">
         <cstring>widgetBaseClassEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="widgetBaseClassEdit">
        <property name="text">
         <string>QWidget</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="pluginClassLabel">
        <property name="text">
         <string>Plugin class &amp;name:</string>
        </property>
        <property name="buddy">
         <cstring>pluginClassEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="pluginClassEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="pluginHeaderLabel">
        <property name="text">
         <string>Plugin &amp;header file:</string>
        </property>
        <property name="buddy">
         <cstring>pluginHeaderEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="pluginHeaderEdit"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="pluginSourceLabel">
        <property name="text">
         <string>Plugin sou&amp;rce file:</string>
        </property>
        <property name="buddy">
         <cstring>pluginSourceEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="pluginSourceEdit"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>Icon file:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="Utils::PathChooser" name="iconPathChooser"/>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="libraryRadio">
          <property name="text">
           <string>&amp;Link library</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QCheckBox" name="skeletonCheck">
          <property name="text">
           <string>Create s&amp;keleton</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="includeRadio">
          <property name="text">
           <string>Include pro&amp;ject</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="descriptionTab">
   <attribute name="title">
    <string>&amp;Description</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>G&amp;roup:</string>
      </property>
      <property name="buddy">
       <cstring>groupEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="groupEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&amp;Tooltip:</string>
      </property>
      <property name="buddy">
       <cstring>tooltipEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="tooltipEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>W&amp;hat's this:</string>
      </property>
      <property name="buddy">
       <cstring>whatsthisEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="whatsthisEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="containerCheck">
      <property name="text">
       <string>The widget is a &amp;container</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="propertyDefaults">
   <attribute name="title">
    <string>Property defa&amp;ults</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTextEdit" name="domXmlEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>dom&amp;XML:</string>
      </property>
      <property name="buddy">
       <cstring>domXmlEdit</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>libraryRadio</tabstop>
  <tabstop>includeRadio</tabstop>
  <tabstop>skeletonCheck</tabstop>
  <tabstop>widgetLibraryEdit</tabstop>
  <tabstop>widgetProjectEdit</tabstop>
  <tabstop>widgetHeaderEdit</tabstop>
  <tabstop>widgetSourceEdit</tabstop>
  <tabstop>widgetBaseClassEdit</tabstop>
  <tabstop>pluginClassEdit</tabstop>
  <tabstop>pluginHeaderEdit</tabstop>
  <tabstop>pluginSourceEdit</tabstop>
  <tabstop>groupEdit</tabstop>
  <tabstop>tooltipEdit</tabstop>
  <tabstop>whatsthisEdit</tabstop>
  <tabstop>containerCheck</tabstop>
  <tabstop>domXmlEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
